integration-test/features/payments.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Create a spontaneous payment
+  Background:
+    Given GPS running
+
+  # Case OK
+  Scenario: An organization creates a spontaneous payment
+    Given the organization "777777" with an enrollment to service "service-1"
+    When the organization creates a spontaneous payment
+    Then the organization gets the created status code 201 
+    And the organization gets the created Payment Position
+    
+  # Case KO: 404 non-existent service
```

Comment:
and the case that organization does't exist ( or is not enabled )  ?

Reply:
At the moment we do not manage the enabled / disabled cases